### ISSUE TITLE
Cannot find 3.6.4 stable

### ISSUE THREAD

**Issue by codeblooded** [pyenv/pyenv#1064]
I'm trying to install 3.6.4, and it's available on python.org. However, it's telling me that I can only install the release candidate:

```
$ pyenv install 3.6.4
python-build: definition not found: 3.6.4

The following versions contain `3.6.4' in the name:
  3.6.4rc1
```

**Comment by yyuu** [comment]
The version has been added in #1058 and you are using some old r revision. Do git pull.

**Comment by codeblooded** [comment]
@yyuu I installed with Homebrew on macOS and it says this is the latest version (pyenv 1.2.0). I'm thinking #1058 has not been distributed through homebrew yet.

**Comment by codeblooded** [comment]
@joshfriend @yyuu I looked back at the commits, and I can confirm that the version packaged currently with homebrew does not contain the fix for 3.6.4.
